src/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/subgraph_builders.hpp

```diff
@@ -4,301 +4,456 @@
 
 #pragma once
 
+#include "ngraph/op/transpose.hpp"
 #include "ngraph_functions/builders.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/assign.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/convolution.hpp"
+#include "openvino/op/gather.hpp"
+#include "openvino/op/lstm_cell.hpp"
+#include "openvino/op/matmul.hpp"
+#include "openvino/op/max_pool.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/non_zero.hpp"
+#include "openvino/op/read_value.hpp"
+#include "openvino/op/relu.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/split.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/op/tensor_iterator.hpp"
+#include "openvino/op/tile.hpp"
 
 namespace ngraph {
 namespace builder {
 namespace subgraph {
-inline std::shared_ptr<ngraph::Function> makeConvPoolRelu(std::vector<size_t> inputShape = {1, 1, 32, 32},
-                                                          ngraph::element::Type_t ngPrc = ngraph::element::Type_t::f32) {
+inline std::shared_ptr<ov::Model> makeConvPoolRelu(std::vector<size_t> inputShape = {1, 1, 32, 32},
+                                                   ov::element::Type_t ngPrc = ov::element::Type_t::f32) {
```

Comment:
can we move implementations to cpp files?
in this case we don't need to keep heavy includes files in `ngraph_functions` headers.

Reply:
Hi. I'd like to do it a little later in scope of [CVS-117139](https://jira.devtools.intel.com/browse/CVS-117139)